core/src/main/scala/spinal/core/Bool.scala

```diff
@@ -58,15 +58,15 @@ class Bool extends BaseType with DataPrimitives[Bool]  with BaseTypePrimitives[B
   private[spinal] override def _data: Bool = this
 
   /**
-    * Logical AND
+    * SpinalHDL logical AND
```

Comment:
Hmm, to be more explicit, i think it should be : 
Hardware logical AND
(same for the ones bellow)